next.config.mjs

```diff
@@ -21,7 +21,7 @@ const walletconnectProjectId = process.env.WALLETCONNECT_PROJECT_ID
 
 export default {
   basePath,
-  webpack5: true,
+  // webpack5: true,
```

Comment:
Why no webpack 5 anymore btw?

Reply:
@manneredboor the Next.js version we use ATM is using Webpack 5 by default. No need to specify Webpack version since Next v12.

Reply:
@manneredboor Next is using Webpack 5 by default since v12, our current version is 12.3.4. 
There were some errors during compilation, complaining on an outdated config schema, so I updated it.